e2e/src/usecases/SecureSend.js

```diff
@@ -1,92 +1,175 @@
-        // for (let index = 0; index < 4; index++) {
-        //   const character = LAST_ACCOUNT_CHARACTERS[index]
-        //   await element(by.id(`SingleDigitInput/digit${index}`)).replaceText(character)
-        // }
-        await element(by.id('ValidateRecipientAccount/TextInput')).replaceText(
-          SAMPLE_WALLET_ADDRESS_VERIFIED_2
+    it('Send cUSD to phone number with multiple mappings', async () => {
+      const commentText = 'test comment old'
+      await waitFor(element(by.id('HomeAction-Send')))
+        .toBeVisible()
+        .withTimeout(30000)
+      await element(by.id('HomeAction-Send')).tap()
+      await waitFor(element(by.id('SendSearchInput')))
+        .toBeVisible()
+        .withTimeout(30000)
+
+      await element(by.id('SearchInput')).tap()
+      await element(by.id('SearchInput')).replaceText(VERIFIED_PHONE_NUMBER)
+      await element(by.id('RecipientItem')).tap()
+
+      // Select the currency
+      await waitFor(element(by.id('cUSDTouchable')))
+        .toBeVisible()
+        .withTimeout(30000)
+      await element(by.id('cUSDTouchable')).tap()
+
+      // Enter the amount and review
+      await element(by.id('SwapInput')).tap()
+      await inputNumberKeypad(AMOUNT_TO_SEND)
+      await element(by.id('Review')).tap()
+
+      // Use the last digits of the account to confirm the sender.
+      await waitFor(element(by.id('confirmAccountButton')))
+        .toBeVisible()
+        .withTimeout(30000)
+      await element(by.id('confirmAccountButton')).tap()
+
+      for (let index = 0; index < 4; index++) {
+        const character = SAMPLE_WALLET_ADDRESS_VERIFIED_2.charAt(
+          SAMPLE_WALLET_ADDRESS_VERIFIED_2.length - (4 - index)
         )
+        await element(by.id(`SingleDigitInput/digit${index}`)).replaceText(character)
+      }
 
-        // Scroll to see submit button
-        await scrollIntoView('Submit', 'KeyboardAwareScrollView', 50)
-        await element(by.id('ConfirmAccountButton')).tap()
+      // Scroll to see submit button
+      await scrollIntoView('Submit', 'KeyboardAwareScrollView', 50)
+      await element(by.id('ConfirmAccountButton')).tap()
 
-        // Write a comment.
-        await addComment(randomContent)
+      // Write a comment.
+      await addComment(commentText)
 
-        // Confirm and input PIN if necessary.
-        await element(by.id('ConfirmButton')).tap()
-        await enterPinUiIfNecessary()
+      // Confirm and input PIN if necessary.
+      await element(by.id('ConfirmButton')).tap()
+      await enterPinUiIfNecessary()
 
-        // Return to home screen.
-        await waitFor(element(by.id('HomeAction-Send')))
-          .toBeVisible()
-          .withTimeout(30 * 1000)
+      // Return to home screen.
+      await waitFor(element(by.id('HomeAction-Send')))
+        .toBeVisible()
+        .withTimeout(30_000)
```

Comment:
nit: consistent number styles. Personally I prefer `30 * 1000`, but `30_000` or `30000` is fine too as long as it's consistent.